### ISSUE TITLE
URL and issue created links need to be updated for user-feedback in GitHub Actions and Issue

### ISSUE THREAD

**Issue by Kremzeeq** [trstringer/manual-approval#126]
Example out from GitHub actions when using the action:
```
Creating issue in repo <Org>/<Repo> with the following content:
Title: Manual approval required for workflow run 9616502346: Deploy Terraform Plan to dev
Approvers: [User]
Body:
Please approve or deny the deployment
Workflow is pending manual review.
URL: https://api.github.com/<Org>/<Repo>/actions/runs/<run-id>[23](https://github.com/<Org>/<Repo>/actions/runs/<run-id>/job/26526189705#step:8:24)46
Required approvers: [User]
Respond "approved", "approve", "lgtm", "yes" to continue workflow or "denied", "deny", "no" to cancel.
Issue created: https://api.github.com/repos/<Org>/<Repo>/issues/9
Workflow status: Pending
```
- URL : When I click on the URL above from the GitHub Actions run, I'm taken to `https://api.github.com/<Org>/<Repo>/actions/runs/<run-id>`. That gives me a 404. I think it's meant to direct to e.g. `https://github.com<Org>/<Repo>/actions/runs/<run-id>/job/26526189705`

- Issue Created:  The link presents some json and doesn't take me to the more human-readable Issues page when I can make the approval. I think it's meant to take me to e.g. `https://github.com/<Org>/<Repo>/issues/9`
So basically, I have to navigate manually to that myself in order to make the approval.

Also, when I navigate to the issue, I'm shown the URL as `https://api.github.com/<Org>/<Repo>/actions/runs/<run-id>` to go and review. I think that also needs updating as that also starts with `https://api.github.com`.

Side note: This is a really innovative GitHub Action your team has worked on by the way. A nice change from manually needing to create environments!

**Comment by cp-yiliu** [comment]
Yes, not only the URL of issue, but the URL of members, e.g. https://api.github.com/orgs/XXXX/teams/XXXXX/members: 404 Not Found. 

And Responding anyone of approval terms doesn't approve the pipeline, the "Workflow Status" is still "pending"

It worked few weeks ago

**Comment by Kremzeeq** [comment]
Ok, keep me updated. I'm happy to test out again

**Comment by virvirlopez** [comment]
hello, any estimated date to fix the issue?

**Comment by moarpheus** [comment]
Bump

**Comment by rdvansloten** [comment]
Bumping this, the Runs link is still dead.

**Comment by lizziemac** [comment]
I believe once this PR is deployed via the action, this issue will be resolved: https://github.com/trstringer/manual-approval/pull/125. I'm running my job off of an image created from main, and am not seeing issues with the action link in the issue

**Comment by lizziemac** [comment]
closing, as is duplicate of issue with more user buy-in https://github.com/trstringer/manual-approval/issues/136
